yiisoft/yii2-bootstrap/CVE-2015-3397.yaml

```diff
@@ -0,0 +1,8 @@
+title:     JSON Data encoded for use in HTML was not safe to use in IE6/IE7, possible XSS attacks
+link:      http://www.yiiframework.com/news/86/yii-2-0-4-is-released/
+cve:       CVE-2015-3397
+branches:
+    master:
```

Comment:
@cebe @pmoust OK. That makes sense. I just marked them on VersionEye. See here: https://www.versioneye.com/php/yiisoft:yii2/2.0.3.